contract/src/main/java/io/neow3j/contract/Nep5Token.java

```diff
@@ -162,10 +167,175 @@ public BigInteger getBalanceOf(Wallet wallet) throws IOException,
         return sum;
     }
 
+    /**
+     * Creates and sends a transfer transaction that uses all accounts in the wallet to cover the amount.
+     * <p>
+     * The default account is used first to cover the amount. If it cannot cover the full amount, the other accounts in
+     * the wallet are iterated one by one to cover the remaining amount. If the amount can be covered, all necessary
+     * transfers are sent in one transaction.
+     *
+     * @param wallet The wallet from which to send the tokens from.
+     * @param to     The script hash of the receiver.
+     * @param amount The amount to transfer as a decimal number (not token fractions).
+     * @return The transaction id.
+     * @throws IOException if there was a problem fetching information from the Neo node.
+     */
+    public NeoSendRawTransaction transferUsingFullWallet(Wallet wallet, ScriptHash to, BigDecimal amount)
```

Comment:
hmm, or even, maybe: `transferFromWalletAddresses()`

Reply:
Alternatively:

`transferFromDefaultAccount` in case only the default account of the wallet should be used
`transferFromSpecificAccounts` in case several specific accounts of the wallet should be used.
`transfer` in case any account of the wallet can be used

Reply:
I like @csmuller proposal!

Reply:
I think we should target those names, then.